lib/determinator/control.rb

```diff
@@ -245,7 +245,8 @@ def has_any_app_version?(required, present)
         !v.match?(Semantic::Version::SemVerRegexp)
       end
       invalid_groups = required.flatten.select do |v|
-        !v.match?(/\d/)
+        without_operator = v.gsub(/>=|>|<=|<|~>/, '')
```

Comment:
Might make sense to do a split instead [like Semantic::Version does](https://github.com/jlindsey/semantic/blob/master/lib/semantic/version.rb#L96)?